mlflow/server/js/src/utils/Utils.js

```diff
@@ -95,7 +95,7 @@ class Utils {
   }
 
   static getGitHubRegex() {
-    return /[@/]github.com[:/]([^/.]+)\/([^/.]+)/;
+    return /[@/]github.com[:/]([^/.]+)\/([^/#]+)#{0,}(.*)/;
```

Comment:
Instead of `#{0,}` it's probably better to write `#*` if you want any number of `#`, or `#?` if you want zero or one. I think `#?` is actually the right thing here because we don't allow multiple `#` characters.

Reply:
Done. Switched to `#?`. Thanks.